cms/templates/js/signatory-editor.underscore

```diff
@@ -0,0 +1,33 @@
+     <div class="signatory-panel-default">
+
+       <% if (!is_editing_all_collections) { %>
+       <a class="signatory-panel-close" href="javascript:void(0);" data-tooltip="Close">
+          <i class="icon fa fa-close" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Close") %></span>
+       </a>
+       <a class="signatory-panel-save"  href="javascript:void(0);" data-tooltip="Save">
+          <i class="icon fa fa-save" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Save") %></span>
+       </a>
+       <% } else if (signatories_count > 1 && (total_saved_signatories > 1 || isNew) ) { %>
+       <a class="signatory-panel-delete" href="#" data-tooltip="Delete">
+          <i class="icon fa fa-trash-o" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Delete") %></span>
+       </a>
+       <% } %>
+       <div class="signatory-panel-header">Signatory <%= signatory_number %></div>
+       <div class="signatory-panel-body">
+         <fieldset class="collection-fields signatory-fields">
+                <div class="input-wrap field text add-signatory-name">
+                    <label><%= gettext("Name ") %></label>
```

Comment:
If the associated ```<input>``` is not a child of the ```<label>``` element, then the label element needs a ```for``` attribute that references the ID of the input.  It's usually easier just to make the input a child.